develop-guide/writeblitz/writeblitz.rst

```diff
@@ -164,6 +177,25 @@ exists in the output. You also, have the option to check if a specific
 
 Both include and exclude keywords are optional to use.
 
+.. note::
+
+    The all the arguments that can be sent to the device while executing a command on a device (specified in this `link
```

Comment:
All the arguments for the execute command can be found at this link.

Reply:
Done